config/crd/bases/topology.nephio.org_nftopologies.yaml

```diff
@@ -165,6 +186,48 @@ spec:
             type: object
           status:
             description: NFTopologyStatus defines the observed state of NFTopology
+            properties:
+              conditions:
+                description: Current service state of the NF.
+                items:
+                  type: string
+                type: array
+              nfInstances:
+                description: Detail on the deployed instances.
+                items:
+                  description: NFDeployedInstance defines an NF instance that is deployed
+                  properties:
+                    clusterName:
+                      description: name of workload cluster where the NF instance
+                        is to be deployed
+                      type: string
+                    connectivities:
+                      description: "TODO(s3wong): the following parameters should
+                        eventually be metadata embedded in a NF package (NFType should
+                        be as well) // NF vendor name NFVendor string `json:\"nfVendor,omitempty\"
+                        yaml:\"nfVendor,omitempty\"` // the software version of this
+                        NF vendor's NFType NFVersion string `json:\"nfVersion,omitempty\"
+                        yaml:\"nfVersion,omitempty\"` \n list of connected NF instances
+                        to this NF instance"
+                      items:
+                        properties:
+                          neighborName:
+                            description: peer NF's Id (see NFInstance struct below)
+                            type: string
+                        type: object
+                      type: array
+                    id:
+                      description: unique ID for this NF instance
+                      type: string
+                    nfType:
+                      description: 'type of NF, example: amf, smf, upf'
```

Comment:
Are quotes required here?

Reply:
the yaml is auto-generated from my comment on the go struct definition --- I don't have quotes in the comment. Probably just regenerate